Tests/AppiumTests/AppiumTests.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import WebDriver
+import XCTest
+
+final class AppiumTests: XCTestCase {
+    private var appiumServerURL: URL? = nil
+
+    override func setUpWithError() throws {
+        super.setUp()
+        appiumServerURL = ProcessInfo.processInfo.environment["APPIUM_SERVER_URL"].flatMap { URL(string: $0) }
+        try XCTSkipIf(appiumServerURL == nil, "APPIUM_SERVER_URL environment variable is not set.")
+    }
+
+#if os(Windows)
+    func testAppium() throws {
+        let webDriver = HTTPWebDriver(endpoint: appiumServerURL!, wireProtocol: .w3c)
+
+        let appiumOptions = Capabilities.AppiumOptions()
+        appiumOptions.automationName = "windows"
+        appiumOptions.app = "Microsoft.WindowsCalculator_8wekyb3d8bbwe!App"
+
+        let capabilities = Capabilities()
+        capabilities.platformName = "windows"
+        capabilities.appiumOptions = appiumOptions
+
+        let session = try Session.createW3C(webDriver: webDriver, alwaysMatch: capabilities)
```

Comment:
it feels a bit duplicative and error prone to specify both `wireProtocol: .w3c` and then also need to create a `Session.createW3C`. Is there a way to unify these concepts?

Reply:
I think so and filed https://github.com/thebrowsercompany/swift-webdriver/issues/183 for it

Reply:
We won't be able to fully eliminate the issue because we'll want one function supporting the W3C `alwaysMatch` + `firstMatch` initialization, and that can't be translated to the legacy protocol. However we can do the conversion the other way around.